src/isar/state_machine/states/docked.py

```diff
@@ -0,0 +1,91 @@
+import logging
+import time
+from typing import TYPE_CHECKING, Optional
+
+from transitions import State
+
+from isar.config.settings import settings
+from isar.models.communication.message import StartMissionMessage
+from isar.services.utilities.threaded_request import (
+    ThreadedRequest,
+    ThreadedRequestNotFinishedError,
+)
+from robot_interface.models.exceptions.robot_exceptions import RobotException
+from robot_interface.models.mission.status import RobotStatus
+
+if TYPE_CHECKING:
+    from isar.state_machine.state_machine import StateMachine
+
+
+class Docked(State):
```

Comment:
If we have a docked state that implies that the robot is not docked in Idle. Should we rename Idle to reflect this?

Reply:
Renamed Docked to IdleDocked to indicate that it is similar to idle

Reply:
Changed to home